tests/test_ciff.py

```diff
@@ -8,7 +8,7 @@ class CiffTest(TempDirTestCase):
     def test_to_and_from_ciff(self):
         from pyterrier_pisa import PisaIndex
         dataset = pt.get_dataset("vaswani")
-        idx = PisaIndex(self.test_dir)
+        idx = PisaIndex(self.test_dir, threads=1)
```

Comment:
why do we now need threads=1 everywhere. Shouldnt threads=1 be default?

Reply:
Fixed, that was for testing for failing tests on github actions. It turns out it wasn't the issue, but forgot to remove.

threads=1 might be a better default, but there are some pros and cons. Wrote up as a separate issue.